app-tasks/dags/aoi-monitoring/find_aoi_projects_to_update.py

```diff
@@ -30,7 +30,7 @@
 dag = DAG(
     dag_id='find_aoi_projects_to_update',
     default_args=default_args,
-    schedule_interval=None,
+    schedule_interval=datetime.timedelta(minutes=5),
```

Comment:
I had thought about whether the runs should be less frequent, but I got hung up on whether the cadence should be thought of as relative or not. Regardless, I pushed a revision that sets the schedule to 6 hours.